frontend/src/metabase/actions/containers/ActionCreator/FormCreator/Description/Description.tsx

```diff
@@ -0,0 +1,28 @@
+import { jt, t } from "ttag";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { getShowMetabaseLinks } from "metabase/selectors/whitelabel";
+import { InfoText } from "./Description.styled";
+
+export function Description() {
+  const docsLink = useSelector(state =>
+    getDocsUrl(state, { page: "actions/custom" }),
+  );
+  const showMetabaseLinks = useSelector(getShowMetabaseLinks);
+
+  return (
+    <InfoText>
+      {jt`Configure your parameters' types and properties here. The values for these parameters can come from user input, or from a dashboard filter.`}
+      {showMetabaseLinks && (
+        <>
+          {" "}
```

Comment:
Some component include left padding, some don't. I erred on the side of simplicity, so I don't have to create a new styled component here, that's why I use a single space.